buildpack.md

```diff
@@ -162,12 +166,38 @@ Executable: `/bin/build <layers[EIC]> <platform[AR]> <plan[ER]>`, Working Dir: `
 
 Using the [Layer Content Metadata](#layer-content-metadata-toml) provided by a buildpack in a `<layers>/<layer>.toml` file, the lifecycle MUST determine:
 
-- Whether the layer directory in `<layers>/<layer>/` should be available to the app (via the `launch` boolean).
-- Whether the layer directory in `<layers>/<layer>/` should be available to subsequent buildpacks (via the `build` boolean).
-- Whether and how the layer directory in `<layers>/<layer>/` should be persisted to subsequent builds of the same OCI image (via the `cache` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be available to the app (via the `launch` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be available to subsequent buildpacks (via the `build` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be persisted and made available to subsequent builds of the same OCI image (via the `cache` boolean).
 
 All combinations of `launch`, `build`, and `cache` booleans are valid. When a layer declares more than one type (e.g. `launch = true` and `cache = true`), the requirements of each type apply.
 
+`build`   | `cache`  | `launch` | Metadata Restored        | Layer Restored      
+----------|----------|----------|--------------------------|---------------------
+T         | T        | T        | Yes - from the app image | Yes - from the cache
+T         | T        | F        | Yes - from the cache     | Yes - from the cache
+T         | F        | T        | No                       | No
+T         | F        | F        | No                       | No
+F         | T        | T        | Yes - from the app image | Yes - from the cache
+F         | T        | F        | Yes - from the cache     | Yes - from the cache
+F         | F        | T        | Yes - from the app image | No
+F         | F        | F        | No                       | No
+
+Notes:
+* The metadata and layer are restored only if they match each other (the cache can be changed/deleted).
```

Comment:
```suggestion
* The metadata and layer are restored only if they match each other.
```
For some reason I find the extra info confusing, but maybe that's just me :/

Reply:
Personally, I think it explains how the metadata and layer can be different.
But I don't have any strong opinion.

Reply:
Hi! I don't understand this sentence, even prior to removing the extra info. By "match each other" I'm presuming it means "the metadata matches the layer contents", but I don't see how the Lifecycle would know whether the two match? For example if the metadata describes the layer as containing a particular version of Node, and the layer contents contains a different version, only the buildpack would have the domain specific knowledge to make that determination, not the Lifecycle.

Which makes me believe this sentence must actually be meaning something else?

Reply:
Hi @edmorley, thank you for your feedback!
I rephrased this sentence. You can see it here: https://github.com/buildpacks/spec/pull/209/commits/3077962a6086ef53124fe502437891b6b90b2049.
Please let me know if you still have any questions.

Reply:
@yaelharel with that edit I understand now - thank you! :-)